src/accounts/accountModel.java

```diff
@@ -0,0 +1,121 @@
+package accounts;
```

Comment:
The User class is actually the model. The intent of this class is to read/write account data, so the code should be in dao.AccountDao with the other classes that read/write data.